docs/foundations/typography/choosing-type/index.md

```diff
@@ -0,0 +1,107 @@
+permalink: /foundations/typography/choosing-type/index.html
+tags:
+  - typography
+subnav:
+  collection: sortedTypography
+  order: 3
+---
+
+## Overview
+
+We use 3 categories of type: Expressive, Editorial, and Utility. Each category is designed for a specific use case.
+
+## Expressive type
+
+Expressive type bridges the gap between our branding and how we use type on the web. It allows for flexibility where type can be used as imagery, use extended weights, and be set in larger sizes. Due to this freeform nature, not only should Expressive type be used very sparingly, <strong>but also very strategically</strong>.
+
+### Type as imagery
+
+Expressive type can be used as imagery if using other assets is not an option or not appropriate. It should feel more like decoration rather than functional text.
+
+<uxdot-example width-adjustment="900px" color-palette="lightest">
+  <img src="./choosing-type-type-as-imagery.svg"
+      alt="examples of expressive type and functional text"
+      width="900"
+      height="395">
+</uxdot-example>
+
+### Extended weights and other fonts
+
+<strong>Expressive type relies on creative direction.</strong> It must represent a specific campaign, event, theme, etc. Extended weights and other fonts provide emphasis and help tell a cohesive story when used in combination with Editorial and Utility text styles.
+
+<uxdot-example width-adjustment="1012px" color-palette="lightest">
+  <img src="./choosing-type-extended-weights-other-fonts.svg"
+      alt="examples of expressive type and functional text"
+      width="1012"
+      height="529">
+</uxdot-example>
+
+### Extra large headings
+
+Expressive type allows for the <strong>extremely limited</strong> use of extra large heading sizes, but not every heading needs to be larger than the sizes in the [scale][scaleandrhythm]. Creative direction or the goal of the page or experience should dictate whether or not heading sizes need to be larger. Follow these best practices as well.
+
+- Use lots of white space around headings
+- Limit the number of words or choose a smaller heading size
+- Maintain a consistent size if extra large headings are used more than once per page or experience
+- Use red or other colors that are part of a theme like Summit
+- Use the bold weight sparingly
+- Line height reduces to 110% or 1.1x the font size 
+
+<uxdot-example width-adjustment="1012px" color-palette="lightest">
+  <img src="./choosing-type-extra-large-headings.svg"
+      alt="examples of text in large font sizes"
+      width="1012"
+      height="543">
+</uxdot-example>
+
+## Editorial type
+
+Editorial type is text written as prose that is readable at both small and large sizes. It can be short- or long-form depending on where it is used, but is subject to change based on the system or user action. Editorial type is very flexible and covers a wide range of use cases, so <strong>use it as the default</strong> to start.
+
+Editorial type includes the following styles.
```

Comment:
Should this end in a colon, like:

`Editorial type includes the following styles:`